src/libs/antares/study/scenario-builder/TSnumberData.cpp

```diff
@@ -307,7 +307,7 @@ bool thermalTSNumberData::reset(const Study& study)
     return true;
 }
 
-void thermalTSNumberData::saveToINIFile(const Study& study, Yuni::IO::File::Stream& file) const
+void thermalTSNumberData::saveToINIFile(const Study&, Yuni::IO::File::Stream& file) const
```

Comment:
If a parameter is passed to a function but not used, I think we should remove it from the definition as well.

Reply:
You are right, for now, parameter **study** restored.